bin/check-node-version

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 const requiredVersion = require( '../package' ).engines.node,
-	semver = require( process.env.SEMVER_GLOBAL_PATH );
+	semver = require( 'semver' );
```

Comment:
I know, that's what this PR rewrite still does. We install semver outside of the general `npm install` process, and check the node version, breaking off `npm install` if it's the wrong version.